applications/crossbar/src/modules/provisioner_v5.erl

```diff
@@ -310,6 +309,82 @@ settings_datetime(JObj) ->
              ),
     wh_json:from_list(Props).
 
+-spec settings_feature_keys(wh_json:object()) -> wh_json:object().
+settings_feature_keys(JObj) ->
+    FeatureKeys = wh_json:get_value([<<"provision">>, <<"feature_keys">>], JObj, wh_json:new()),
+    Brand = get_brand(JObj),
+    Family = get_family(JObj),
+    AccountId = wh_json:get_value(<<"pvt_account_id">>, JObj),
+    wh_json:foldl(
+        fun(Key, Value, Acc) ->
+            Type = wh_json:get_binary_value(<<"type">>, Value),
+            V = wh_json:get_binary_value(<<"value">>, Value),
+            FeatureKey = get_feature_key(Type, V, Brand, Family, AccountId),
+            maybe_add_feature_key(Key, FeatureKey, Acc)
+        end
+        ,wh_json:new()
+        ,FeatureKeys
+    ).
+
+-spec get_feature_key(ne_binary(), wh_json:object(), ne_binary(), ne_binary(), ne_binary()) -> wh_json:object().
```

Comment:
Wrong spec. `Value` isn't a JSON object, is it? And `Brand` isn't ne_binary(). And `Family` isn't ne_binary()